tensorboard/webapp/angular_polymer_bridge/BUILD

```diff
@@ -0,0 +1,10 @@
+load("//tensorboard/defs:defs.bzl", "tf_ts_library")
```

Comment:
We already have tb_polymer_interop_types.  No need to redefine the abstractions here.

Reply:
I was trying to keep functionality out of a "types" file, but thinking about it more, that does make sense